easybuild/easyconfigs/u/UCX/UCX-1.15.0-rc4-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'ConfigureMake'
+
+name = 'UCX'
+version = '1.15.0-rc4'
```

Comment:
Are we OK with using a release candidate here? There's already an `rc6`, but we should probably stick to 1.14.1 instead?